core/src/main/java/com/predic8/membrane/core/interceptor/balancer/LoadBalancingInterceptor.java

```diff
@@ -131,7 +131,7 @@ public Outcome handleResponse(Exchange exc) {
             }
 
             if (sessionId != null) {
-                balancer.addSession2Cluster(sessionId, BalancerUtil.getSingleClusterNameOrDefault(balancer), exc.getProperty("dispatchedNode", Node.class));
+                balancer.addSession2Cluster(sessionId, BalancerUtil.getSingleClusterNameOrDefault(balancer), exc.getPropertyOrNull("dispatchedNode", Node.class));
```

Comment:
when `exc.getPropertyOrNull("dispatchedNode", Node.class)` returns `null`, this leads to problems, or won't it?